pytorch_lightning/metrics/classification/accuracy.py

```diff
@@ -63,10 +82,19 @@ class Accuracy(Metric):
         >>> accuracy(preds, target)
         tensor(0.5000)
 
+        >>> target = torch.tensor([0, 1, 2])
+        >>> preds = torch.tensor([[0.1, 0.9, 0], [0.3, 0.1, 0.6], [0.2, 0.5, 0.3]])
+        >>> accuracy = Accuracy(top_k=2)
+        >>> accuracy(preds, target)
+        tensor(0.6667)
+
     """
+
     def __init__(
         self,
         threshold: float = 0.5,
+        top_k: Optional[int] = None,
+        mdmc_accuracy: str = "subset",
```

Comment:
To be backward compatible the default should be `global`:
```suggestion
        mdmc_accuracy: str = "global",
```